changelog.d/2075.breaking.rst

```diff
@@ -0,0 +1 @@
+Stop recognizing files ending with ``.dist-info`` as distribution metadata
```

Comment:
Do you want this to be a breaking change? A changelog around a breaking change should include directions as to how a user should respond to account for the breakage. If there's no reason to think there's any action required by the user, then I would just make this a regular 'change'.

Reply:
To expedite things, I'm going to change this to a 'change'.